### ISSUE TITLE
Multi-threading Trios raw fails

### ISSUE THREAD

**Issue by raphidoc** [nasa/HyperCP#225]
When multi-threading Trios raw, only one file at a time is passed to the worker. 
This can't work as each instruments have it's own file. Instead, pass a list of three coincident files.

**Comment by oceancolorcoder** [comment]
Right. I had forgotten that detail. Let me point you to some HyperOCR data instead.

**Comment by oceancolorcoder** [comment]
@raphidoc On the Box, look under Data/KORUS/. I've added a few more files to test the multithreading.
